modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountEntryDisplay.java

```diff
@@ -32,18 +32,18 @@
 public class AccountEntryDisplay {
 
 	public static AccountEntryDisplay of(AccountEntry accountEntry) {
-		return new AccountEntryDisplay(accountEntry);
+		if (accountEntry != null) {
+			return new AccountEntryDisplay(accountEntry);
+		}
+
+		return _EMPTY_INSTANCE;
 	}
 
 	public static AccountEntryDisplay of(long accountEntryId) {
 		AccountEntry accountEntry =
 			AccountEntryLocalServiceUtil.fetchAccountEntry(accountEntryId);
 
-		if (accountEntry != null) {
-			return new AccountEntryDisplay(accountEntry);
-		}
-
-		return _EMPTY_INSTANCE;
+		return AccountEntryDisplay.of(accountEntry);
```

Comment:
SF is complaining about this, since you can just use `of` without attaching it to the class. You can also probably just inline the fetch call instead of making the `accountEntry` variable.

Reply:
Ah ok. Updated the commit with the change. Thanks!